v3/client.go

```diff
@@ -115,7 +120,7 @@ func newInternal(dynamoDB DynamoDBClient, tableName string, opts ...ClientOption
 
 // New creates a new dynamoDB based distributed lock client.
 func New(dynamoDB DynamoDBClient, tableName string, opts ...ClientOption) (*Client, error) {
-	internalClient, err := newInternal(dynamoDB, tableName, opts...)
+	internalClient, err := newInternal(dynamoDB, tableName, "", opts...)
```

Comment:
Perhaps replace the literal with a constant with an intention revealing name? `noSortKeyName`?

Reply:
Sam actually suggested exactly this to me, and I recommended `""` 😂. To my eyes, `""` is the [zero value](https://go.dev/ref/spec#The_zero_value) for strings, and is sufficient. If I saw `noSortKeyName` I'd wonder whether there was some magic string like `"__NO_SORT_KEY__"` that needed to be used!

I could be convinced either way, though.

Reply:
Seeing `noSortKeyName` in the caller here, would definitely be a little clearer than a mystery empty string…

Reply:
> Seeing noSortKeyName in the caller here, would definitely be a little clearer than a mystery empty string…

I agree